src/components/boardComponent/boardComponent.js

```diff
@@ -2,29 +2,23 @@ import React, { Component } from 'react';
 import Square from '../squareComponent/squareComponent';
 import GameHeader from '../gameHeaderComponent/gameHeaderComponent';
 import './BoardComponent.css';
+import BoardState from '../../boardState/boardState';
 
 class Board extends Component {
```

Comment:
This could be renamed to `BoardComponent` or `BoardView` to make it less ambiguous which is the view and which is the data model